snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DefinitionSemanticIT.java

```diff
@@ -16,48 +16,42 @@
 import dev.ikm.tinkar.terms.TinkarTerm;
 import org.junit.jupiter.api.Test;
 
+import java.io.IOException;
 import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class DefinitionSemanticIT extends BaseIntegrationTest {
-
-
-    @Test
-    protected void testDefinitionSemantics() {
-        executeTestLogic();
-    }
+public class DefinitionSemanticIT extends AbstractIntegrationTest {
 
     /**
      * Test Definition Semantics.
      *
      * @result Reads content from file and validates Definition of Semantics by calling private method assertDefinition().
      */
-    @Override
-    public void executeTestLogic() {
+    @Test
+    public void testDefinitionSemantics() throws IOException {
         String sourceFilePath = "../snomed-ct-origin/target/origin-sources/SnomedCT_ManagedServiceUS_PRODUCTION_US1000124_20240901T120000Z/Full/Terminology/sct2_TextDefinition_Full-en_US1000124_20240901.txt";
         String errorFile = "target/failsafe-reports/descriptions_definitions_not_found.txt";
-        int notFound = 0;
-        // When
+        AtomicInteger notFound = new AtomicInteger(0);
 
-        processFileLines(sourceFilePath, columns -> {
-            UUID id = UuidT5Generator.get(UUID.fromString("3094dbd1-60cf-44a6-92e3-0bb32ca4d3de"), columns[0]); //Need hardcode ID on namespace for Snomed
-            long effectiveTime = SnomedUtility.snomedTimestampToEpochSeconds(columns[1]);
-            StateSet descriptionStatus = Integer.parseInt(columns[2]) == 1 ? StateSet.ACTIVE : StateSet.INACTIVE;
-            EntityProxy.Concept descriptionType = SnomedUtility.getDescriptionType(columns[6]);
-            String term = columns[7];
-            EntityProxy.Concept caseSensitivityConcept = SnomedUtility.getDescriptionCaseSignificanceConcept(columns[8]);
+        processFile(sourceFilePath, errorFile);
```

Comment:
You want to assign "notFound" to the return type here and you can turn "notFound" back to an int.